sdk/src/connections/usb/windows/usb_depth_sensor_windows.cpp

```diff
@@ -662,6 +674,18 @@ aditof::Status UsbDepthSensor::readAfeRegisters(const uint16_t *address,
                                                 uint16_t *data, size_t length) {
     using namespace aditof;
 
+    if (address == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters address null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    if (data == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters data null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    assert(length > 0);
```

Comment:
`length` has the type `size_t` which by definition is unsigned, so this is not needed.

Reply:
In this way we will prevent the length to be set to 0.